tests/integration/targets/vmware_appliance_info/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+vcenter_port: 443
```

Comment:
vcenter_port: 1080

Reply:
the variables in defaults/main.yml should be used for eco vcenter and they are overwritten by variable in [vars.yml](https://github.com/ansible-collections/vmware.vmware/pull/29/files#diff-575f9dc8e37da616efa732c1272fe792777998e5eea595078cff3ed17e3f65d0R6) when the role is imported from run.yml playbook. 

It looks like in the execution the port it's using is indeed 1080 but it fails:

```
TASK [vmware_appliance_info : Gather appliance info] ***************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to connect to vCenter or ESXi API at 127.0.0.1:1080 : 404 Client Error: Not Found for url: https://127.0.0.1:1080/api"}
```